code/modules/client/preferences.dm

```diff
@@ -28,6 +28,7 @@ datum/preferences
 	var/UI_style_alpha = 255
 	var/motd_hash = ""					//Hashes for the new server greeting window.
 	var/memo_hash = ""
+	var/safety_check = FALSE //Checks if you are on non-harm when firing. Defaults off.

```

Comment:
This change appears to be pointless with the swap to `parallax_togs`.

Reply:
Shit shit shit

Reply:
resolved